app/selectors/view.js

```diff
@@ -1,3 +1,3 @@
 import Immutable from 'immutable';
 
-export const getViewState = (state, viewId) => state.getIn(['dataView', viewId]) || new Immutable.Map();
+export const getViewState = (state, viewId) => Immutable.fromJS(state).getIn(['dataView', viewId]) || new Immutable.Map();
```

Comment:
[Here](https://github.com/rasendubi/heutagogy-chrome-extension/blob/a64b4119683b3b47d593568fa716d1833ed16c6e/app/components/LoginForm/LoginForm.js#L133) we're already passing immutable state, why do we need to call fromJS second time?